encoders/image/PaddleHubEncoder/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM jinaai/jina:devel
+
+# setup the workspace
+COPY . /workspace
+WORKDIR /workspace
+
+RUN apt-get update && \
+    apt-get -y install libgomp1 libgl1-mesa-glx libglib2.0-0
+
+# install the third-party requirements
+RUN pip install -r requirements.txt && pip uninstall -y pathlib
+
+# for testing the image
+RUN pip install -y pytest pytest-mock mock && JINA_TEST_PRETRAINED='true' pytest
```

Comment:
are we sure the `JINA_TEST_PRETRAINED` test is run? 

Shouldn't it be `export JINA_TEST_PRETRAINED=true`?

Reply:
my bad, actually it seems to work this way

Reply:
Yes alternatively ```ENV JINA_TEST_PRETRAINED='true'``` should propagate the environment variable, setting it in the entire context. Export wasn't getting recognised.